stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -676,70 +566,55 @@ end
 #! Input: [PK, MSG, ...]
 #! Output: [...]
```

Comment:
Could we also add a description here of what we expect to be on the advice stack? Maybe we could use something similar to the format used [here](https://github.com/0xPolygonMiden/miden-base/blob/main/crates/miden-lib/asm/kernels/transaction/lib/prologue.masm#L95).

Reply:
Added